src/core/SIP/SIPHeader.cs

```diff
@@ -2250,7 +2266,11 @@ public new string ToString()
                 {
                     foreach (var authHeader in AuthenticationHeaders)
                     {
-                        headersBuilder.Append(authHeader.ToString() + m_CRLF);
+                        var value = authHeader.ToString();

```

Comment:
When ``SIPDigest=null``, the original logic here will generate unexpected blank lines.